app/models/metric/capture.rb

```diff
@@ -234,4 +238,5 @@ def self.minutes_ago(value)
       value.to_i_with_method.seconds.ago.utc
     end
   end
+  private_class_method :minutes_ago
```

Comment:
I am personally not a fan of `class << self` notation because when the body of that block becomes more than a screen, you lose the visual context and think it's an instance method.  Would prefer the private_class_method approach at least for now.

Moving forward, if there are a lot of private_class_methods that are related, then that is a smell for something that should probably be extracted to an object with instance methods and a proper public/private API.